Lib/test/test_grammar.py

```diff
@@ -917,56 +917,59 @@ def g3():
         check_syntax_error(self, "class foo:return 1")
 
     def test_break_in_finally(self):
-        count = 0
-        while count < 2:
-            count += 1
-            try:
-                pass
-            finally:
-                break
-        self.assertEqual(count, 1)
+        with warnings.catch_warnings():
+            warnings.simplefilter('ignore', SyntaxWarning)
```

Comment:
How does this fix work? The SyntaxWarning is issued when the file is imported so this block hasn't run yet

Reply:
Right. I'll refix this.